tests/element_test_vectors.rs

```diff
@@ -664,79 +740,64 @@ mod native_element_tests {
             &[]
         }
 
-        fn make_reader() -> Self::ElementReader {
+        fn make_reader() -> Self::ReaderApi {
             native_element_reader()
         }
-
-        // TODO: These methods return an ion-c writer for the time being.
-        //       Once we have a native ElementWriter impl and the reader is passing (most of) its
-        //       ion-tests, we can modify this.
-        fn make_text_writer(buffer: &mut [u8]) -> IonResult<SliceElementWriter> {
-            Format::Text(TextKind::Compact).element_writer_for_slice(buffer)
-        }
-
-        fn make_pretty_text_writer(buffer: &mut [u8]) -> IonResult<SliceElementWriter> {
-            Format::Text(TextKind::Pretty).element_writer_for_slice(buffer)
-        }
-
-        fn make_binary_writer(buffer: &mut [u8]) -> IonResult<SliceElementWriter> {
-            Format::Binary.element_writer_for_slice(buffer)
-        }
     }
 
     #[test_resources("ion-tests/iontestdata/good/**/*.ion")]
     #[test_resources("ion-tests/iontestdata/good/**/*.10n")]
     fn native_good_roundtrip_text_binary(file_name: &str) {
-        good_roundtrip_text_binary(NativeElementApi {}, file_name)
```

Comment:
🗺️ Originally, `NativeElementApi` was defined like this:

```rust
struct NativeElementApi {}
```

This PR changed it to:
```rust
struct NativeElementApi;
```

which means that all usages can now drop their `{}`s.